react/features/token-data/User.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Implements a user of conference.
+ */
+export default class User {
```

Comment:
I agree but this class comes from legacy and we use this interface in legacy code.